sdk/api_fingerprint_impl.go

```diff
@@ -0,0 +1,170 @@
+package sdk
+
+import (
+	"context"
+	"encoding/json"
+	"io"
+	"net/http"
+	"net/url"
+)
+
+type FingerprintApiService struct {
+	cfg *Configuration
+}
+
+func (f *FingerprintApiService) makeRequest(ctx context.Context, requestUrl *url.URL, method string) (*http.Request, error) {
+	request, err := http.NewRequest(method, requestUrl.String(), nil)
+
+	if err != nil {
+		return request, err
+	}
+
+	handleAuth(ctx, request)
+
+	request.Header.Add("User-Agent", f.cfg.UserAgent)
+
+	return request, nil
+}
+
+func (f *FingerprintApiService) path(definition requestDefinition, params ...string) string {
+	return f.cfg.basePath + definition.Path(params...)
```

Comment:
nitpick: `definition.Path` doesn't look like a function, it looks like a field. I'd rather call it `definition.GetPath` or `definition.CreatePath`

Reply:
Done!